### ISSUE TITLE
Youtube URL Decryption Fails

### ISSUE THREAD

**Issue by theScrabi** [TeamNewPipe/NewPipe#2012]
Please see  [Youtube URL Decryption Fails #133](https://github.com/TeamNewPipe/NewPipeExtractor/issues/133)

**Comment by trymeouteh** [comment]
Yeah as of today NewPipe will not load any videos.

**Comment by TheMenda** [comment]
Confirmed. Now it's 100% broken.

**Comment by jusdepatate** [comment]
Version 0.14.1 works for me 🤔
![screenshot_20190118-070123](https://user-images.githubusercontent.com/29034627/51368254-ec451600-1aee-11e9-8c84-0dee58f1c862.jpg)

**Comment by jusdepatate** [comment]
It is unable to load some videos but most are working

**Comment by CrowsVeldt** [comment]
I'm having the same thing on Xiaomi Note4, NewPipe version 14.2. I'll try downgrading as well to see if it helps.

**Comment by stienz** [comment]
Samsung Note 9 here. Downgrading to 14.1 lets me load videos again.

**Comment by dH0rs3** [comment]
It's working in Version 14.1.

**Comment by shyby** [comment]
Can confirm that 14.2 not working and 14.1 working

**Comment by sscoderss** [comment]
Do you have any solution for this problem?

**Comment by FaizKhan5** [comment]
May be revert some changes can fix this which is in 0.14.1 😊

**Comment by MR4Y** [comment]
@CrowsVeldt You have to uninstall the latest version before downgrading. Android will not let you replace an app with an earlier version of the same app.

**Comment by CrowsVeldt** [comment]
@MR4Y Thanks

**Comment by CrowsVeldt** [comment]
Yeah, 14.1 worked for me for about half an hour before it started failing like before. I hope the issue will be resolved without too much trouble. I don't have much programming experience but if there's something simple I could do to help I'd be happy to

**Comment by ondernuray** [comment]
I installed 014.1 and it works well. You can see crash report.

Alıcı: crashreport@newpipe.schabi.org | Alıcı: crashreport@newpipe.schabi.org
-- | --
Alıcı: crashreport@newpipe.schabi.org

**Comment by B0rlaug** [comment]
@ondernuray  your crash report isn't needed as the issue has already been fixed, please wait until the next release for the fix (and other PR's) to be added, thanks.

**Comment by wisor12** [comment]
> The issue was fixed here: [https://github.com/TeamNewPipe/NewPipeExtractor/pull/134](url) and the next update will be released soon enough.
> 
> Please be patient as both @theScrabi and @TobiGr are busy at the moment and please do not open new issues asking about the next release date, cheers.

Thanks a bunch as always for your quick work guys!
Just a quick follow up question because it would be nonsensical to open another thread/issue case.

I still can't add my subscriptions no matter what I try and believe me I tried everything and more. (downloading the subscription_manager.xml via my desktop, some mild editing etc.)

My guess is NewPipe still has issues with special characters/symbols within the sub names. (I follow japanese/russian channels with Kanji/Cyrillic in it)

That or it can't handle a high subscription number (subscribed to 551 channels)

**Comment by theScrabi** [comment]
All right, so thanks to the people at [this thread over at the newpipe extractor](https://github.com/TeamNewPipe/NewPipeExtractor/issues/133), and the excellent work of the youtube-dl team we now have a fix.

**Comment by mintyleaf** [comment]
well, the brand new 0.15 still having this issue :c

**Comment by theScrabi** [comment]
So here it is version [v0.15.0](https://github.com/TeamNewPipe/NewPipe/releases/tag/v0.15.0).

**Comment by Iolaum** [comment]
I just installed the v0.15.0 apk, unfortunately the issue still persisted in my device as well.

**Comment by akks1981** [comment]
not fixed "Could not decrypt video url signature" with 0.15.0

**Comment by theScrabi** [comment]
gradle didn't chache the new extractor when building. Fixed now. try downloading v0.15.0 again.

**Comment by Iolaum** [comment]
Updated apk works! Thank you for the prompt response!!

**Comment by akks1981** [comment]
> 
> 
> gradle didn't cache the new extractor when building. Fixed now. try downloading v0.15.0 again.

now it works BIG THX

**Comment by mouarfff** [comment]
> 
> 
> @CrowsVeldt You have to uninstall the latest version before downgrading. Android will not let you replace an app with an earlier version of the same app.

WIth fdroid you can downgrade version
